app/views/articles/_actions.html.erb

```diff
@@ -51,35 +51,35 @@
             class="crayons-link crayons-link--block"
             rel="noopener"
             href='https://twitter.com/intent/tweet?text=<%= u "\"#{@article.title.strip}\" by " %><%= u @article.user.twitter_username ? "@#{@article.user.twitter_username}" : @article.user.name %><%= u " #{Settings::General.twitter_hashtag} " %><%= u " #{article_url(@article)}" %>'>
-            Share to Twitter
+            <%= t("core.share_to", media: "Twitter") %>
```

Comment:
I think it's ok for `share_to` to stay under `core` directly but we could also consider a namespace like `article_actions`. It may be best to add that a bit later though.

Reply:
The reactions will be included in this article_actions namespace?

Reply:
Let's leave it as-is for now, sometimes it's easier to make these calls when more examples exist. Thanks for double-checking!